luigi/parameter.py

```diff
@@ -223,6 +227,23 @@ def parse(self, x):
         """
         return x  # default impl
 
+    def parse_list(self, xs):
```

Comment:
They're public because they're used by Task. is_batchable gets called in Task.batch_param_names and parse_list gets called in Task.from_str_params